src/backend/joanie/core/models/base.py

```diff
@@ -54,20 +59,38 @@ def class_name(self):
         """
         return self._meta.model_name
 
-    def get_cache_key(self, prefix, is_language_sensitive=False, language=None):
+    def get_cache_key(self, prefix=None, is_language_sensitive=False, language=None):
         """
         Return a cache key for the instance. If the key is going to be used for multilingual
         content, an extra argument `is_local_sensitive` can be set to True to bind
```

Comment:
nitpicking and out of scope but:
```suggestion
        content, an extra argument `is_language_sensitive` can be set to True to bind
```